internal/fakecgo/doc.go

```diff
@@ -28,3 +28,6 @@
 // Purego will pick which ever Cgo runtime is available and prefer the one that
 // comes with Go (runtime/cgo).
 package fakecgo
+
+//go:generate go run gen.go
+//go:generate go fmt symbols.go
```

Comment:
`go fmt -w symbols.go`

Reply:
Ah, I found `gofmt` and `go fmt` are different. I usually do

```
gofmt -s -w symbols.go
```

What do you think?

Reply:
`go fmt` doesn't support `-s` so I replaced it with `gofmt` directly.

Reply:
Didn't see you had commented! I agree tho